jre_emul/jre_sources.mk

```diff
@@ -1680,6 +1680,7 @@ JAVA_PRIVATE_SOURCES_ZIP = \
   java/util/zip/ZStreamRef.java \
   java/util/zip/ZipCoder.java \
   java/util/zip/ZipConstants64.java \
+  java/util/zip/ZipUtils.java \
```

Comment:
Change to com/google/j2objc/util/ZipUtils.java and move to its alphabetical location, per above comments.

Reply:
changed file path